app/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepository.scala

```diff
@@ -58,12 +58,12 @@ class UnusedApplicationsRepository @Inject()(mongo: ReactiveMongoComponent)(impl
       background = true)
   )
 
-  def applicationsByEnvironment(environment: Environment): Future[List[UnusedApplication]] = find("environment" -> environment)
+  def unusedApplications()(implicit environment: Environment): Future[List[UnusedApplication]] = find("environment" -> environment)
```

Comment:
does environment need to be implicit here? if its used in the query would it make more sense to have it as an argument to the function?